debian/changelog

```diff
@@ -2,9 +2,10 @@ cuttlefish-common (0.9.8) UNRELEASED; urgency=medium
 
   * Changes to enable arm builds
   * Fix dependencies for buster
-  * Add qemu-user-static on non-amd64
```

Comment:
this removal seems wrong?

Reply:
and I'm not convinced about the add in the next line either... did someone previously misgenerate this or???